backend/flask_api.py

```diff
@@ -199,5 +199,55 @@ def search():
         return (str(e), 400)
 
 
+# Search function which gets a text and does a full text search on all attributes
+@app.route('/api/projects/search/simple/<int:offset>/<int:count>/', methods=['GET'])
```

Comment:
This is obviously not going to be a trivial fix, since the pagination is using these functions, though it should not be that big of a deal.
I am currently trying to make it work with the updated api, but the search is only returning empty arrays form me atm. GET /api/projects returns the Projects properly.
Perhaps I am not handling it correctly, but I am sort of stuck here, because I don't have a clue what could be causing this behavior.